lib/jxl/dec_external_image.cc

```diff
@@ -281,6 +290,8 @@ Status ConvertToExternal(const jxl::ImageBundle& ib, size_t bits_per_sample,
   Image3F temp_color, unpremul;
   const ImageF* alpha = ib.HasAlpha() ? &ib.alpha() : nullptr;
   ImageF temp_alpha;
+  const ImageF* extra = &ib.extra_channels()[extra_channel_index];
```

Comment:
`extra = want_extra_channel ? ... : nullptr;`  extra_channel_index is not relevant if want_extra_channel is false and even if you specify that it must be 0 it could be possible that extra_channels is empty.

Reply:
No longer an issue after https://github.com/libjxl/libjxl/pull/325